scripts/analyze-cover-profile

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+MINIMUM_TEST_COVERAGE = 76.7
+
+# coverprofile.out (see Makefile for how this is generated) will look something like this:
+#
+# github.com/aws/amazon-ecs-agent/agent/wsclient/error.go:66:						Retry						0.0%
+# github.com/aws/amazon-ecs-agent/agent/wsclient/errors.go:23:						Error						0.0%
```

Comment:
nit: formatting?

Reply:
that's how the output actually looks, they are tabs so i guess how it lines up depends on how large github's tab spacing is.